soh/soh/OTRGlobals.cpp

```diff
@@ -118,6 +122,8 @@ extern "C" void InitOTR() {
     InitCosmeticsEditor();
     DebugConsole_Init();
     Debug_Init();
+    Rando_Init();
+    InitItemTracker();
```

Comment:
looks like github collapsing some comments made this one easy to miss @briaguya-ai

Reply:
the item tracker and the main randomizer menu are separate imgui windows. i guess we could call over to the init from `randomizer_item_tracker` in `randomizer`, but i'm not sure if that is clearer. also, while the item tracker is currently in the randomizer menu, it works when playing vanilla as well.

i think it's worth considering where the item tracker should live instead of tying it in to randomizer even more.

Reply:
Got it! Ok, currently its tied to the randomizer by name.. but makes sense to leave it separate in case we decide to move it out and then it should probably be renamed and moved away from the randomizer folder.